common/src/api/external/mod.rs

```diff
@@ -2741,13 +2957,30 @@ mod test {
         let net_des = serde_json::from_str::<IpNet>(&ser).unwrap();
         assert_eq!(net, net_des);
 
+        let net_str = "fd00:99::1/64";
+        let net = IpNet::from_str(net_str).unwrap();
+        let ser = serde_json::to_string(&net).unwrap();
+
+        assert_eq!(format!(r#""{}""#, net_str), ser);
+        let net_des = serde_json::from_str::<IpNet>(&ser).unwrap();
+        assert_eq!(net, net_des);
+        println!("{:?} = {:?}", net, net_des);
```

Comment:
Nit, but this information shows up if the assertion is hit above.

Reply:
Removed.